spec/controllers/downloads_controller_spec.rb

```diff
@@ -5,10 +5,10 @@
   describe "with a file" do
     let(:depositor) { FactoryGirl.find_or_create(:archivist) }
     let(:file) do
-      GenericFile.new.tap do |f|
+      GenericFile.create do |f|
         f.apply_depositor_metadata(depositor.user_key)
-        f.add_file(File.open(fixture_path + '/world.png'), 'content', 'world.png')
-        f.save!
+        f.label = 'world.png'
```

Comment:
Label was a fedora 3 builtin. We've replaced it with a simple predicate. `add_file` in hydra-head used to set that label, but we've deprecated that module so now we're doing it manually.